tests/std/tests/VSO_0000000_allocator_propagation/test.cpp

```diff
@@ -34,6 +34,155 @@ _CONSTEXPR20 void assert_is_permutation(const Container& cont, initializer_list<
     assert(is_permutation(cont.begin(), cont.end(), il.begin(), il.end()));
 }
 
+class test_leak {
+    char* ptr;
+
+public:
+    test_leak(const test_leak&)            = delete;
+    test_leak& operator=(const test_leak&) = delete;
+
+    _CONSTEXPR20 test_leak() noexcept /* terminates */ : ptr(allocator<char>{}.allocate(1)) {}
+    _CONSTEXPR20 ~test_leak() {
+        allocator<char>{}.deallocate(ptr, 1);
+    }
+};

```

Comment:
This works well with constexpr basic_string. I'm really confused what happens with the `vector(vector&&)` in `static_assert(test_sequence<vector>)`'s `test_sequence_move_ctor`